### DISCUSSION TITLE
How to deploy my next.js app to Strato?

### DISCUSSION THREAD

**Discussion by MiladUchiha** [vercel/next.js#57824] [Help]
### Summary

Hi!
I bought a domain from Strato and I don't have any idea how to deploy my next.js app. I've tried next build and uploaded the out folder but I just get the frontend
I'm a beginner in deploying 
I appreciate any help
Thanks!

**Comment by lucianmurmurache** [comment]
This may not be the answer you are looking for but I will leave a comment here just in case it is, I have a project where the domain and email service were purchased via strato(mail plus), I deployed everything to vercel and then enabled the DNS, which I added to Strato in the domain settings (Domains > Domainverwaltung > Settings icon > DNS ). You have to update the NS and A records, just select "Eigene IP-Adresse" and add your info there. Hope this helps!
